src/commands/user/info.ts

```diff
@@ -65,15 +61,18 @@ export default class User {
       embed.setColor(genRGBColor(r, g, b) as ColorResolvable);
     } catch { }
 
-    if (memberRoles.length != 0) {
-      embed.addFields({
-        name: `🎭 • ${guildRoles.filter((role) => selectedMember.roles.cache.has(role.id)).size - 1} ${rolesOrRole}`,
-        value: `${memberRoles
-          .slice(0, 5)
-          .map((role) => `<@&${role[1].id}>`)
-          .join(", ")}${memberRoles.length > 5 ? ` **and ${memberRoles.length - 5} more**` : ""}`,
-      });
-    }
+    const guildRoles = guild.roles.cache.filter(role => selectedMember.roles.cache.has(role.id));
+    const memberRoles = [...guildRoles].sort((role1, role2) => role2[1].position - role1[1].position);
+    memberRoles.pop();
+    const rolesOrRole = memberRoles.length === 1 ? "role" : "roles";
```

Comment:
Why even put this in a variable? To be honest this looks a bit gooff...

Reply:
idk next commit has it removed now